MovieApp/LogInViewController.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  LogInViewController.swift
+//  MovieApp
+//
+//  Created by Petar on 23.03.2023..
+//
+
+import Foundation
+import UIKit
+import PureLayout
+
+class LoginViewController: UIViewController {
+    
+    let backgroundColor = UIColor(red: 0.04, green: 0.14, blue: 0.247, alpha: 1.0)
+    let textFieldColor = UIColor(red: 0.07, green: 0.23, blue: 0.39, alpha: 1.0)
+    let buttonColor = UIColor(red: 0.30, green: 0.69, blue: 0.87, alpha: 1.0)
+    
+    private var signInLabel: UILabel!
+    private var emailAdressLabel: UILabel!
+    private var usernameField: UITextField!
+    private var passwordLabel: UILabel!
+    private var passwordField: UITextField!
+    private var signInButton: UIButton!
+    
+    private var screenHeight: CGFloat!
+    private var screenWidth: CGFloat!
+    
+    override func viewDidLoad() {
+        
+        super.viewDidLoad()
+        view.backgroundColor = backgroundColor
+        
+        createViews()
+        customizeViews()
+        defineViewLayout()
+        
+    }
+    
+    
+    override var supportedInterfaceOrientations: UIInterfaceOrientationMask {
+        return .portrait
+    }
+
+    override var shouldAutorotate: Bool {
+        return true
+    }
+    
+    
+    private func createViews() {
+        
+        screenHeight = view.frame.height
+        screenWidth = view.frame.width
+
+        
+        signInLabel = UILabel(frame: CGRect(x: 0, y: screenHeight*0.15, width: screenWidth, height: 35))
```

Comment:
Please use the `UILabel()` constructor and use constraints to position this label and give it its size.